src/index.js

```diff
@@ -0,0 +1,169 @@
+  img.src = imgURL;
+}
+
+
+const displaySky = document.getElementById('sky');
+const dropdownValue = document.getElementById('weather-dropdown');
+const changeSky = (skyDrop) => {
+  if (skyDrop === 'sun') {
+    return '☁️ _ ☁️ _ _ _ ☀️ _ _ _ ☁️ _ ☁️';
+  } else if (skyDrop === 'clouds') {
+    return '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+  } else if (skyDrop === 'rain') {
+    return '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+  } else if (skyDrop === 'snow') {
+    return '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+  } else {
+    return ' ';
+  }
+};
+
+
+
+
+dropdownValue.addEventListener('change', function handleChange () {
+  let newSky = changeSky(dropdownValue.options[dropdownValue.selectedIndex].value);
+  displaySky.textContent = `${newSky}`;
+  display
+});
+
+logTempText();
+changeTextColor();
+logLandscape(temperatureNumber);
+
+const currentWeather = document.getElementById('current-temp-button');
+currentWeather.addEventListener("click", function () {
+  if (textBox.value !== '') {
+    findLocation(textBox.value);
+  } else {
+    findLocation('San Diego, CA, USA');
+  }
+});
+
+const upButton = document.getElementById('increase-temp-button');
+upButton.textContent = '🔥';
+upButton.addEventListener("click", function() {
+  temperatureNumber += 1;
+  logTempText();
+  changeTextColor();
+  logLandscape(temperatureNumber);
+});
+
+const downButton = document.getElementById('decrease-temp-button');
+downButton.innerText = '❄️';
+downButton.addEventListener("click", function() {
+  temperatureNumber -= 1;
+  logTempText();
+  changeTextColor();
+  logLandscape(temperatureNumber);
+});
+
+const resetCity = document.getElementById('reset-city-button');
+resetCity.addEventListener("click", function() {
+  currentCity = 'San Diego, CA, USA';
+  document.getElementById('display-city').textContent = currentCity;
+  document.getElementById('text-box').value='';
+})
+
+const textBox = document.getElementById('text-box');
+textBox.onkeyup = function () {
+  document.getElementById('display-city').innerHTML = textBox.value;
+};
+
+
+/*   ~~CALLING APIs~~   */
+// const axios = require('axios');
+
+const findLocation = (location) => {
+  axios.get('http://127.0.0.1:5000/location',
+  {
+    params: {
+      q: location,
+    }
+  })
+  .then( (response) => {
+    lat = response.data[0].lat;
+    long = response.data[0].lon;
+    console.log('Succesful location call!')
+
+    let secondCall = findWeather(lat,long);
+    secondCall;
+    console.log('Second weather call!');
+  })
+  .catch( (error) => {
+    console.log('ERROR')
+  });
+}
+
+const findWeather = (lat, long) => {
+  axios.get('http://127.0.0.1:5000/weather',
+  {
+    params: {
+      lat: lat,
+      lon: long,
+    }
+  })
+  .then( (response) => {
+    temperatureResponse = Math.floor(response.data['main']['temp']);
+    // weatherResponse = JSON.stringify(response.data['weather'][0]['main']);
+    // console.log(`Successful weather response: ${weatherResponse}`)
+    temperatureNumber = temperatureResponse
```

Comment:
I noticed you aren't converting kelvin to farenheit which throws off your `changeLandscape` logic since you're using farenheit to determine which landscape to display.